exercises/concept/poetry-club-door-policy/.docs/instructions.md

```diff
@@ -96,6 +96,6 @@ backDoorResponse('Stands so high   ');
 Change the implementation of the `backDoorPassword` function so that it's polite:
 
 ```javascript
-backDoorPassword('esroh');
+backDoorPassword('horse');
```

Comment:
I think `esroh` was right.

Reply:
if `esroh` was right then the exemplar and the test suite are wrong because neither of those expect the string to be reversed: https://github.com/exercism/javascript/blob/main/exercises/concept/poetry-club-door-policy/.meta/exemplar.js#L61-L63

Reply:
Ha yes. Of course. Stupid.

It _does_ reverse it, but it does that as part of the test suite. aka, the _guards_ give you the sentences reversed, and `backDoorResponse` picks the _reversed_ first letter, which are then passed into `backdoorPassword`. So by the time they get to this, it's no longer reversed.

TL;DR: you're right.